.vitepress/sidebars/concepts.ts

```diff
@@ -1,69 +1,59 @@
 import { DefaultTheme } from "vitepress";
 
 export const conceptsSidebar: DefaultTheme.SidebarItem[] = [
```

Comment:
Run this file in prettier, it should be the default formatter?

Reply:
Will do.

Reply:
I mistakenly thought this comment was about running prettier in index.ts but I just solved the merge conflict (as far as I can tell).